packages/semantic-tokens/etc/semantic-tokens.api.md

```diff
@@ -0,0 +1,84 @@
+## API Report File for "@fluentui/semantic-tokens"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export const ctrlFocusOuterStroke: string;
+
+// @public (undocumented)
+export const ctrlLinkForegroundBrandHover: string;
+
+// @public (undocumented)
+export const ctrlLinkForegroundBrandHoverRaw = "--ctrl-link-foreground-brand-hover";
```

Comment:
We shipped prefixing our tokens with `--smtc-`, we shoudl probably align on that.

Reply:
I'm a little concerned this isn't matching the JSON I've received, it's easy enough for us to add but need to be sure we are all synced on this - can discuss this at our meeting today.